src/javascript/EditPanel/header/UpperSection.jsx

```diff
@@ -76,9 +59,25 @@ export const HeaderUpperSection = ({title, actionContext}) => {
                 </div>
 
                 <div className={styles.headerRight}>
+                    <div className={styles.saveActions}>
+                        <DisplayActions context={{
```

Comment:
Can we have several actions here ? Can't we use DisplayAction (without "s") instead

Reply:
Yes, we can have multiple actions here event if it's not a button group. Romain wanted that.